sdk/modelparser/azure-digitaltwins-parser/src/main/java/com/azure/digitaltwins/parser/implementation/codegen/JavaLine.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.parser.implementation.codegen;
+
+import java.io.IOException;
+
+public class JavaLine extends JavaStatement {
+
+    private String text;
+
+    /**
+     * Initializes a new instance of the {@link JavaLine} class.
```

Comment:
having the @link here to JavaLine, wouldn't that just cause a navigation to the same page? so no-op?

Reply:
Yeah, I use the link so the documentation would change if I decide to refactor the class name.